config/locales/server.en.yml

```diff
@@ -4,6 +4,9 @@ en:
     discourse_reactions_reaction_for_like: "Defines the reaction associated to to the like action."
     discourse_reactions_enabled: "Enable the discourse-reactions plugin."
     discourse_reactions_enabled_reactions: "Defines a list of enabled reactions, any emoji is allowed here."
+    placeholder:
+      discourse_reactions_enabled_reactions:
+        newEmoji: "Enter Emoji Name"
```

Comment:
we don't use snakeCase, should be new_emoji

Reply:
where is this used btw ?

Reply:
It was used in the placeholder of textfield which we removed yesterday